### ISSUE TITLE
Should I install this software with "sudo" option?

### ISSUE THREAD

**Issue by TsuruMasato** [jrl-umi3218/mc_openrtm#18]
Hello, this is Masato Tsuru in JRL, AIST, Japan.

I am trying to install this software from source by using
`$ccmake ..` + `$ make -j8` + `$ make install` commands.

However, the installation process seems to go into /opt/ros/~~ directory.
In my understanding, all mc_rtc and related components are installed into somewhere(/home/openrtp? I don't know exactly...) without sudo option.
Actually, I checked that my CMAKE_INSTALL_PREFIX in mc_openrtm is surely set to "/home/masato/openrtp", which is same as the other AIST JRL libraries' installation location.

And in my case, mc_openrtm is installed by the most useful script `build_and_install.sh` in mc_rtc.
I should not run it with sudo option, I think.

![Screenshot from 2022-07-05 19-21-45](https://user-images.githubusercontent.com/14979823/177307242-4f49fe98-9872-490d-b665-51be78f1bba8.png)

In conclusion, should I use sudo command to install mc_openrtm into my system?
Or is there any option to specify another location to install?

**Comment by gergondet** [comment]
Hi @TsuruMasato 

mc_openrtm will detect hrpsys-base installation automatically and install files. It seems to find an installation in ROS which is maybe not the one you want to target.

Because it is using pkg-config to find hrpsys-base you can try two things:
- modify PKG_CONFIG_PATH so that your openrtp installation is first
- modify PKG_CONFIG_PATH to hide the ROS path (but it might break ROS detection for mc_rtc)

**Comment by TsuruMasato** [comment]
Thanks to your advice, maybe I could fix it!!
I changed the order of environment path, to search openrtp directory first.

Actually, I am now using RTM-ROS bridge to pass virtual PointCloud (in Chorenoid) from RTM topic to ROS topic.
So I have ros-melodic-hrpsys-base and some related apt packages in my /opt/ros/melodic directory.
It caused that mis-direction in package config.

![Screenshot from 2022-07-06 15-07-38](https://user-images.githubusercontent.com/14979823/177480632-20bd9d92-8576-48f6-9284-8b40524a63c1.png)

If there were no reactions from you, I couldn't understand the problem.
Thank you so much.
